src/Spout/Common/Entity/Style/CellVerticalAlignment.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Box\Spout\Common\Entity\Style;
+
+/**
+ * Class CellVerticalAlignment
+ * This class provides constants to work with text vertical alignment.
+ */
+abstract class CellVerticalAlignment
+{
+    public const BOTTOM = 'bottom';
+    public const CENTER = 'center';
+    public const DISTRIBUTED = 'distributed';
+    public const JUSTIFY = 'justify';
+    public const TOP = 'top';
```

Comment:
According to the ODS spec, here are the valid values:
- auto
- baseline
- bottom
- middle
- top

You have a special transform for "center" (=> "middle"), but what about the others (distributed/justify)?

Reply:
I think as there are a completely different set of valid values for vertical alignment for XLSX and ODS respectively, all we can easily do here is allow all of those values to be valid, but still with the transformation of 'center' to 'middle' for ODS, so this is what I've done. Unless you have another suggestion?